plugin/pkg/auth/authorizer/node/node_authorizer_test.go

```diff
@@ -50,6 +51,7 @@ func init() {
 }
 
 func TestAuthorizer(t *testing.T) {
+	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.TokenRequest, true)()
```

Comment:
see usage of features field in table test for toggling features during this test

Reply:
Changed this to match how CSI feature flag is used.